torch/onnx/_internal/fx/passes/assertion_removal.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import annotations
+
+import torch
+from torch.onnx._internal import _beartype
+from torch.onnx._internal.fx import _pass
+
+
+class RemoveAssertions(_pass.Transform):
+    """This pass removes all assertion and check nodes from the FX graph."""
+
+    _ATEN_ASSERTION_TARGETS = {
+        torch.ops.aten.sym_constrain_range_for_size.default,
+        torch.ops.aten._assert_async.msg,
+    }
+
+    @_beartype.beartype
```

Comment:
We don’t really need beartype here because the method is not even typed. Moving forward we should only use beartype very selectively where it provides the most value